Assignment/solution_student/css/style.css

```diff
@@ -27,15 +27,179 @@ html {
 .clearfix:after {
   clear: both;
 }
-/******************************************
-/* BASE STYLES
-/*******************************************/
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
+  color: #05080C;
+  line-height: 1.5;
+  max-width: 900px;
+  margin: 70px auto;
+  background: #F6F6F6;
+  border-top: 5px solid #222222;
+  font-family: 'open sans',sans-serif;
+  border-bottom: 5px solid #222222;
+  padding-top: none;
+}
+
+header {
+    height: 100px;
+    margin:0 auto;
+    padding:5px;
+    background:#003366;
+    text-align: left center;
```

Comment:
`text-align` only excepts one value, so you need to choose either `left` or `center` :)